tests/controllers/test_rss_controller.py

```diff
@@ -180,16 +176,6 @@ def test_try_parse_feed_with_beautiful_soup(feed_input: str, is_url: bool, enabl
     assert isinstance(feed_data["entries"], list)
 
 
-@pytest.mark.parametrize("enable_cache", [True, False])
-def test_enable_disable_cache(enable_cache):
-    controller = RSSController(feed_input=test_urls, enable_cache=enable_cache)
-
-    if enable_cache:
-        assert controller.session_exist
-    else:
-        assert not controller.session_exist
-
-
```

Comment:
Lets add an example which would trigger the redundant fetch in the old code but works now.